### PR TITLE
Fixed documentation typos in DeviceGroup sample class

### PR DESCRIPTION
**Pull request by erwink**
## References

References #xxxx

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @erwink,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by raboof** [comment]
OK TO TEST

File: akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java [hunk 1]
```diff
@@ -91,7 +91,7 @@ private void onTrackDevice(DeviceManager.RequestTrackDevice trackMsg) {
     } else {
       log.warning(
           "Ignoring TrackDevice request for {}. This actor is responsible for {}.",
-          groupId,
+              trackMsg.groupId,
```
Comment (raboof) [inline akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java]: Good observation! I suspect our style checker will complain about the extra indentation here, though...

**Review by raboof — APPROVED** [review]
Thanks! Probably needs a formatting fix before it is accepted by Jenkins.

File: akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java (hunk 1 above)
Comment (erwink) [reply akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java]: Thanks. What's next step?

**Review by chbatey — APPROVED** [review]
LGTM, run a scalafmt and then jenkins should pass

File: akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java (hunk 1 above)
Comment (patriknw) [reply akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java]: @erwink Would you be able to run
```
sbt akka-docs/scalafmtAll`
```
and commit the changes and push to your branch again?

**Comment by erwink** [comment]
Fixed docs sample

**Comment by erwink** [comment]
Could you please send me the command I need to run to eventually push that.

File: akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java (hunk 1 above)
Comment (erwink) [reply akka-docs/src/test/java/jdocs/tutorial_4/DeviceGroup.java]: @patriknw Please send me step by step command I need to run to push eventually push.

**Comment by patriknw** [comment]
git commit
git push origin akka-docs-device-sample-fix

**Comment by erwink** [comment]
Hi Patrik,

Done.
Let me know if it works for you.

Thanks,
Erwin

**Comment by raboof** [comment]
I don't think they got pushed (there is no new commit at https://github.com/erwink/akka/commits/akka-docs-device-sample-fix). Did you successfully make a local commit with the formatting changes? Can you show the output of what happens when you attempt to push them?

**Comment by raboof** [comment]
(if you prefer we can take care of it as well)

**Comment by erwink** [comment]
git push origin master
warning: not sending a push certificate since the receiving end does not
support --signed push
Total 0 (delta 0), reused 0 (delta 0)
To https://github.com/erwink/akka.git
   071ea34eb1..2517ff8595  master -> master

**Comment by raboof** [comment]
you wanted to push to your `akka-docs-device-sample-fix` branch, though, right? So it'd be `git push origin akka-docs-device-sample-fix` (while in your local akka-docs-device-sample-fix branch).

You can use 'git log' to verify if you successfully added a commit with your formatting changes.

**Comment by erwink** [comment]
git commit
On branch akka-docs-device-sample-fix
Your branch is up to date with 'origin/akka-docs-device-sample-fix'.

nothing to commit, working tree clean

**Comment by erwink** [comment]
Please note my last commit includes the fix was on Apr 2.

**Comment by raboof** [comment]
Ah, I think I understand the confusion now: after your contribution on Apr 2nd, we approved the change but mentioned you need to format the source code, which is easiest with `scalafmt`.

You asked for "the command I need to run to eventually push that". We assumed you successfully formatted the code and were having trouble pushing - but as I understand now you haven't formatted the code yet?

The command for formatting would be `sbt akka-docs/scalafmtAll`. Sorry for the confusion!

**Comment by erwink** [comment]
Done successfully.
sbt akka-docs/scalafmtAll

Any other thing should be done?

Thanks,
Erwin

**Comment by raboof** [comment]
Now commit and push the changes made by scalafmt - that should make them show up at https://github.com/erwink/akka/commits/akka-docs-device-sample-fix and in this PR. If all goes well we can merge after that! Thanks for hanging in there!

**Comment by erwink** [comment]
Something still weird:

git commit -m "document typo fix"
On branch akka-docs-device-sample-fix
Your branch is up to date with 'origin/akka-docs-device-sample-fix'.

nothing to commit, working tree clean
$ git push origin akka-docs-device-sample-fix
Everything up-to-date

**Comment by raboof** [comment]
```
git commit -m "document typo fix"
On branch akka-docs-device-sample-fix
Your branch is up to date with 'origin/akka-docs-device-sample-fix'. 
```
This suggests your changes have not yet been 'added'.

Committing changes to git is a 2-step process: you first 'add' them, and then 'commit'. You can use 'git status' and 'git diff' to inspect the outstanding changes. If you're happy, you can add them with 'git add' and then commit with 'git commit'. If you're confident everything can be committed, you can skip the 'git add' and use 'git commit -a' to commit all pending changes.

If you'd like to have more background on how all of this works, a git tutorial like https://www.codecademy.com/learn/learn-git might be useful to look into. For now I suspect the above might be sufficient for now ;)

**Comment by erwink** [comment]
I familiar with git I have done add command.
Let me check again.

**Comment by erwink** [comment]
As I see here : https://github.com/akka/akka/pull/26661/commits I have one
pending commit.

**Comment by raboof** [comment]
Thanks @erwink!

**Comment by erwink** [comment]
You welcome.
Thank you.

Cheers,
Erwin

**Pull request merged**